lib/src/revset.rs

```diff
@@ -91,8 +91,9 @@ fn resolve_full_commit_id(
         }
         let commit_id = CommitId::new(binary_commit_id);
         match repo.store().get_commit(&commit_id) {
-            Ok(_) => Ok(Some(vec![commit_id])),
-            Err(BackendError::ObjectNotFound { .. }) => Ok(None),
+            // Only recognize a git commit_id if it is known to jj
```

Comment:
Since this code is not specific to Git, let's write it something like "Only recognize a commit if we've indexed it"

Reply:
Done.